plugins/search-react/src/components/SearchTracker/SearchTracker.tsx

```diff
@@ -23,14 +23,18 @@ import { useSearch } from '../../context';
  */
 export const TrackSearch = ({ children }: { children: React.ReactChild }) => {
   const analytics = useAnalytics();
-  const { term } = useSearch();
+  const { term, result } = useSearch();
+
+  const numberOfResults = result.value?.numberOfResults ?? 0;
```

Comment:
I wonder if we should differentiate between `0`, meaning "This search query definitely resulted in 0 results" vs. `undefined`, meaning "It's unknown how many results this query returned (either because the search engine does not support it, or because permissions are enabled, etc)"

Reply:
Yep done.